src/kudu/consensus/quorum_util.cc

```diff
@@ -893,6 +893,7 @@ void AdjustVoterDistributionWithCurrentVoters(
     const RaftConfigPB& config,
     std::map<std::string, int> *voter_distribution) {
   CHECK(voter_distribution);
+
```

Comment:
Probably left from earlier commits? Looks like there are no changes needed in this file anymore.